splash/defaults.py

```diff
@@ -13,9 +13,9 @@
 VIEWPORT_SIZE = '1024x768'
 
 # Window size limitations.
-VIEWPORT_MAX_WIDTH = 20000
-VIEWPORT_MAX_HEIGTH = 20000
-VIEWPORT_MAX_AREA = 4000*4000
+VIEWPORT_MAX_WIDTH = 40000
+VIEWPORT_MAX_HEIGTH = 40000
+VIEWPORT_MAX_AREA = 1280*VIEWPORT_MAX_HEIGTH  # support max height @ 1280 width
```

Comment:
a nitpick: could you please add spaces around `*`, as per pep8?

Reply:
Sure! I didn't put them because I thought it was the standard for the project.

Reply:
Done